salt/utils/vault.py

```diff
@@ -126,24 +126,28 @@ def _get_vault_connection():
         return _get_token_and_url_from_master()
 
 
-def make_request(method, resource, profile=None, **args):
+def make_request(method, resource, token=None, vault_url=None, get_token_url=False, profile=None, **args):
```

Comment:
Can you move these new kwargs to the end fo the list? We don't want to cause an API change and cause regressions for calls to this function.